helm/cert-exporter/templates/daemonset-policy-exceptions.yaml

```diff
@@ -0,0 +1,42 @@
+{{- if .Capabilities.APIVersions.Has "kyverno.io/v2alpha1/PolicyException" }}
```

Comment:
Reminder: this means that if this app is installed outside `kube-system`, there is a chance Pods will be blocked on restart if this app is installed before kyverno. The risk is fine with me for now since we're working on better ways of managing exceptions, but important for the app owners to know.